SPT-767 Made Mitigating validFrom datetime

chris-dco opened the pull request:
## What

Updated MitigatingCredentialClass.validFrom field to `datetime`.

## Why

CIMIT is required to produce `validFrom` in ISO time and date format. So the `validFrom` field needs to be changed to datetime.

chrisclayson reviewed (dismissed):
Wish your IDE would stop re-formatting things ... granted just getting rid of whitespace.

galund reviewed (changes requested):
Please at least make the whitespace changes in a different commit otherwise you break the git history for those lines. Right now it's not actually clear to me what this change is actually _doing_.

chris-dco committed:
SPT-767 Made Mitigating validFrom datetime

CIMIT is required to produce validFrom is ISO time and date format.
So the validFrom field needs to be changed to datetime.

galund reviewed (commented):
Can we get an example put into the `v1/examples` directory, and then get that validated in the tests?

Then we'd probably catch this kind of thing sooner - and we can easily check that the example matches the one in the RFC over here: https://github.com/govuk-one-login/architecture/blob/main/rfc/0041-CI-mitigation-VC.md

thanks!

chris-dco committed:
SPT-767 Added sample for SecurityCheck and test
